uwrt_utils/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(uwrt_utils)
+
+# Load catkin and package dependencies
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  std_msgs
+)
+
+# Declare a catkin package
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
+  CATKIN_DEPENDS
+  roscpp
+  std_msgs
+)
+
+# Specify additional locations of header files
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+)
+
+# Declare and link the C++
+add_library(${PROJECT_NAME}
+  src/uwrt_can.cpp
+  src/${PROJECT_NAME}_can_test_node.cpp
```

Comment:
what does that mean lel?

Reply:
catkin is just a bunch of CMAKE macros. there are macros dedicated to declaring code that should only need to be compiled if you specify u want to compile tests. theres also a way to mark dependencies to only pull test dependencies if u want them. there's also ways to specifically marks certain targets as tests so that they run when you call `catkin test`

Reply:
like unit test style

Reply:
i say we do this later

Reply:
Ya like I said, super low priority, beyond me caring level. Open a ticket tho.

Reply:
made issue #62